app/models/permitted_params.rb

```diff
@@ -303,8 +303,9 @@ def project(instance = nil)
   end
 
   def time_entry
-    params.fetch(:time_entry, {}).permit(:hours, :comments, :work_package_id,
-                                       :activity_id, :spent_on, custom_field_values: [])
+    params.fetch(:time_entry, {}).permit(
+      :hours, :comments, :work_package_id, :activity_id, :spent_on,
+      custom_field_values: custom_field_keys)
```

Comment:
As I said the question is if we actually want to whitelist or not. This is not whitelisting but accepting anything at all for custom field values. But seeing as this is already done it's ok then I guess.